docs/java-api/query-dsl/percolate-query.asciidoc

```diff
@@ -19,15 +19,14 @@ a document containing a percolator query should be indexed:
 --------------------------------------------------
 // create an index with a percolator field with the name 'query':
 client.admin().indices().prepareCreate("myIndexName")
-                        .addMapping("query", "query", "type=percolator")
```

Comment:
the percolator field (`query`) is still needed ?

Reply:
Oops, thanks! I wonder why the test is still passing (will check this out).